src/rules/radixRule.ts

```diff
@@ -50,13 +50,24 @@ function walk(ctx: Lint.WalkContext<void>) {
             (
                 // parseInt("123")
                 isIdentifier(node.expression) && node.expression.text === "parseInt" ||
-                // window.parseInt("123") || global.parseInt("123")
+                // window.parseInt("123") || global.parseInt("123") || Number.parseInt("123")
                 isPropertyAccessExpression(node.expression) &&
                 node.expression.name.text === "parseInt" &&
                 isIdentifier(node.expression.expression) &&
                 (
                     node.expression.expression.text === "global" ||
-                    node.expression.expression.text === "window"
+                    node.expression.expression.text === "window" ||
+                    node.expression.expression.text === "Number"
+                ) ||
+                // window.Number.parseInt("123") || global.Number.parseInt("123")
+                isPropertyAccessExpression(node.expression) &&
+                node.expression.name.text === "parseInt" &&
+                isPropertyAccessExpression(node.expression.expression) &&
+                node.expression.expression.name.text === "Number" &&
+                isIdentifier(node.expression.expression.expression) &&
+                (
+                    node.expression.expression.expression.text === "global" ||
+                    node.expression.expression.expression.text === "window"
```

Comment:
please split this absolute unit of a condition into multiple variables, one for each `||` clause. 
so it reads like `if (isParseInt || isGlobalParseInt || isNumberParseInt)` instead of this madness which is quite hard to validate correctness now that you've added more `||` clauses.